stage-1/checkout-pr-branch/action.yml

```diff
@@ -14,11 +14,9 @@ runs:
   using: composite
   steps:
     - name: Configure git
+      working-directory: ${{ github.action_path }}
       shell: bash
-      run: |
-        git config --global user.name "github-actions[bot]"
-        git config --global user.email "41898282+github-actions[bot]@users.noreply.github.com"
-        git config push.default current
+      run: ../../configure-git/configure-git.sh
 
```

Comment:
I would have tried reusing the action like
```yaml
- name: Configure git
  uses: ${{ github.action_path }}/../../configure-git
```
but apparently `uses:` [doesn't support contexts](https://github.com/actions/runner/issues/1348#issuecomment-921180705) (in this case the `github` context is needed)